pyproject.toml

```diff
@@ -1,2 +1,2 @@
 [tool.black]
-target-version = ["py37", "py38"]
+target-version = ["py38"]
```

Comment:
You could remove the `.coveragerc` file and put it's contents in here like so

```toml

[tool.coverage.run]
relative_files = true

[tool.coverage.report]
precision = 2
show_missing = true
skip_covered = true
exclude_lines = ["if __name__ == .__main__.:", "def __str__", "def __repr__", "pragma: no cover"]
```

you can also put isort contents in here as well, but it doesn't look like flake8 supports project.toml. Putting the converage config in here at least lets you drop the other config file.

Reply:
I put isort and flake8 into the setup.cfg file

Reply:
yeah I think it's fine to put some in setup.cfg and some in pyproject.toml. It's annoying to me that all these tools haven't adopted pyproject.toml though.